nix/cells/cloud/library.nix

```diff
@@ -0,0 +1,8 @@
+{ cell
+, inputs
+}: {
+  actions = {
```

Comment:
I don't really understand the purpose of this file. What difference does this make? Comments please!

Reply:
Seems like we can just hard-code those strings in, which I would be in favour of.

Reply:
Do so if you want, that's fine with me. This is just to extract some common strings to variables to avoid possible future typos.

Reply:
That's fine, but worth a comment. I also don't know why these magic strings are what they are :grimacing:

Reply:
They are just arbitrary names used to refer to the input fact in the tullia task.